inc/Engine/Support/Data.php

```diff
@@ -61,15 +62,34 @@ public function __construct( Options_Data $options ) {
 	 * @return array
 	 */
 	public function get_support_data() {
-		$active_options = array_intersect_key( $this->to_send, array_filter( $this->options->get_options() ) );
-
 		return [
 			'Website'                  => home_url(),
 			'WordPress Version'        => get_bloginfo( 'version' ),
-			'WP Rocket Version'        => rocket_get_constant( 'WP_ROCKET_VERSION' ),
+			'WP Rocket Version'        => rocket_get_constant( 'WP_ROCKET_VERSION', '' ),
 			'Theme'                    => wp_get_theme()->get( 'Name' ),
 			'Plugins Enabled'          => implode( ' - ', rocket_get_active_plugins() ),
-			'WP Rocket Active Options' => implode( ' - ', $active_options ),
+			'WP Rocket Active Options' => implode( ' - ', $this->get_active_options() ),
 		];
 	}
+
+	/**
+	 * Returns the active options
+	 *
+	 * @return array
+	 */
+	public function get_active_options(): array {
+		$active_options = array_intersect_key( $this->to_send, array_filter( $this->options->get_options() ) );
```

Comment:
We didn't have any feedback related to that from the support, I figure this never has been an issue for them. And this uses the global values, not page by page like the meta. I want to avoid doing to many requests to the DB as much as possible.